Makefile

```diff
@@ -124,13 +125,13 @@ bin dist:
 	@mkdir -p $@
 
 image: Containerfile $(BINARIES)
-	@buildah manifest rm ghcr.io/openbao/$(PLUGIN_PREFIX)-$(PLUGIN):$(VERSION) || true
-	@buildah manifest create ghcr.io/openbao/$(PLUGIN_PREFIX)-$(PLUGIN):$(VERSION)
-	@$(foreach target,$(TARGETS),cat $< | PLUGIN=$(PLUGIN) envsubst '$$PLUGIN' | buildah build -f - --platform $(subst _,/,$(target)) --build-arg PLUGIN=$(PLUGIN) -t $(PLUGIN_PREFIX)-$(PLUGIN):$(VERSION)_$(target);)
-	@$(foreach target,$(TARGETS),buildah manifest add ghcr.io/openbao/$(PLUGIN_PREFIX)-$(PLUGIN):$(VERSION) $(PLUGIN_PREFIX)-$(PLUGIN):$(VERSION)_$(target);)
+	@buildah manifest rm $(REGISTRY)/$(PLUGIN_PREFIX)-$(PLUGIN):$(VERSION) || true
+	@buildah manifest create $(REGISTRY)/$(PLUGIN_PREFIX)-$(PLUGIN):$(VERSION)
+	@$(foreach target,$(TARGETS),cat $< | PLUGIN=$(PLUGIN) envsubst '$$PLUGIN' | buildah build -f - --platform $(subst _,/,$(target)) --build-arg PLUGIN=$(PLUGIN) -t $(REGISTRY)/$(PLUGIN_PREFIX)-$(PLUGIN):$(VERSION)_$(target);)
+	@$(foreach target,$(TARGETS),buildah manifest add $(REGISTRY)/$(PLUGIN_PREFIX)-$(PLUGIN):$(VERSION) $(REGISTRY)/$(PLUGIN_PREFIX)-$(PLUGIN):$(VERSION)_$(target);)
 
 push: image
-	@buildah manifest push ghcr.io/openbao/$(PLUGIN_PREFIX)-$(PLUGIN):$(VERSION)
+	@buildah manifest push --all $(REGISTRY)/$(PLUGIN_PREFIX)-$(PLUGIN):$(VERSION)
```

Comment:
Ahhh yes, I figured this one out the hard way as well locally in the past, when having to re-push tags up to our other registries.